python/py-crossbar/Portfile

```diff
@@ -0,0 +1,188 @@
+                    port:py${python.version}-click \
+                    port:py${python.version}-constantly \
+                    port:py${python.version}-cryptography \
+                    port:py${python.version}-h2 \
+                    port:py${python.version}-idna \
+                    port:py${python.version}-incremental \
+                    port:py${python.version}-jinja2 \
+                    port:py${python.version}-lmdb \
+                    port:py${python.version}-mistune \
+                    port:py${python.version}-netaddr \
+                    port:py${python.version}-passlib \
+                    port:py${python.version}-priority \
+                    port:py${python.version}-psutil \
+                    port:py${python.version}-pyubjson \
+                    port:py${python.version}-asn1 \
+                    port:py${python.version}-asn1-modules \
+                    port:py${python.version}-pygments \
+                    port:py${python.version}-pynacl \
+                    port:py${python.version}-openssl \
+                    port:py${python.version}-pyqrcode \
+                    port:py${python.version}-pytrie \
+                    port:py${python.version}-yaml \
+                    port:py${python.version}-sdnotify \
+                    port:py${python.version}-service_identity \
+                    port:py${python.version}-setproctitle \
+                    port:py${python.version}-setuptools \
+                    port:py${python.version}-treq \
+                    port:py${python.version}-txaio \
+                    port:py${python.version}-txtorcon \
+                    port:py${python.version}-u-msgpack-python \
+                    port:py${python.version}-urllib3 \
+                    port:py${python.version}-watchdog \
+                    port:py${python.version}-werkzeug \
+                    port:py${python.version}-zlmdb \
+                    port:py${python.version}-zopeinterface
+}
+
+subport crossbar {
+    name                crossbar
+    version             ${version}
+    revision            0
+    categories          net
+
+    distfiles
+    patchfiles
+
+    description         Run the crossbar.io WAMP application router as a \
+                        service.
+    long_description    {*}${description}
+
+    depends_build
+    depends_lib         port:py-crossbar
+
+    set cb_config_path  ${prefix}/etc/${name}
+    set cb_data_path    ${prefix}/var/db/${name}
+    set cb_log_path     ${prefix}/var/log/${name}
+    set cb_share_path   ${prefix}/share/${name}
+
+    set cb_plist_path   ${prefix}/etc/LaunchDaemons/org.macports.${name}
+
+    set cb_config_file  ${cb_config_path}/config.json
+    set cb_config_src   ${cb_share_path}/config.json.example
+    set cb_log_file     ${cb_log_path}/crossbar.log
+
+    set cb_plist        org.macports.${name}.plist
+
+    set cb_bin          ${prefix}/bin/crossbar-${python.branch}
+
+    set cb_user         crossbar
```

Comment:
We've been following a strategy Apple introduced in Mac OS X 10.5 where service usernames and groupnames are prefixed with an underscore. Unless Apple has changed gears and no longer recommends that strategy, we should follow it here too (e.g., `_crossbar`).

Reply:
Changed